shared/templates/accounts_password/ansible.template

```diff
@@ -5,7 +5,31 @@
 # disruption = low
 - (xccdf-var var_password_pam_{{{ VARIABLE }}})
 
-- name: Ensure PAM variable {{{ VARIABLE }}} is set accordingly
+{{% if product == "ol8" %}}
+- name: {{{ rule_id }}} - Find pwquality.conf.d files
+  find:
+    paths: /etc/security/pwquality.conf.d/
+    patterns: "*.conf"
+  register: pwquality_conf_d_files
+
+- name: {{{ rule_id }}} - Ensure {{{ VARIABLE }}} is not set in pwquality.conf.d
```

Comment:
Please, to keep it aligned to other rules, use {{{ rule_title }}} instead of {{{ rule_id }}}